services/hosted_services.md

```diff
@@ -1,19 +1,91 @@
+
+## Add-ons
+Aeon has an add-on architecture implemented in [LUA](https://www.lua.org/). Add-ons can be added at the individual client level or on the server. Client add-ons are incorporated into our Aeon client install in our staff workstation image and replicated on every client instance. Server add-ons are installed in a single place within the ILLiad server application. See available [add-ons](https://atlas-sys.atlassian.net/wiki/spaces/ILLiadAddons/pages/3149603/Aeon+Addon+Directory).
+
+### Server Add-ons we use.
+* [ReCAP Add-On](https://github.com/PrincetonUniversityLibrary/aeon_scsb_addon) - Facilitates transactions with SCSB
+
+### Getting Support 
+General documentation is available on the [Atlas website](https://support.atlas-sys.com/hc/en-us/categories/360000720853-Aeon). Support can be requested through the [Atlas Web Portal](https://support.atlas-sys.com/hc/en-us/requests/new)
+
+## Using the client for troubleshooting/account management
+We have several desktop machines that are available for remote desktop connection that will allow IT staff who don't use windows to utilize. Once connected you need to use actual Aeon client staff credentials to login into the various clients. 
+
+* lib-ares-cli1
+* lib-ares-cli2
+* lib-ares-cli3
+
+## ARES
+Ares is used manage reading lists of Library materials. Students enrolled in course via reading lists and follow links to the full content of Library materials from those lists. Faculty and other staff involved in supporting those course use Ares to add new materials to course reading lists. Library staff also add items to reading lists on request. Review recent [releases](https://support.atlas-sys.com/hc/en-us/articles/360011824074-Ares-Release-Schedule). 
+
+The materials in these lists primarily link to library electronic content hosting on our vendor resourses but they do include links to materials in the Library Catalog, Figgy and Finding Aids. They also contain links to digitized videos stored in the [Video Reserves application](https://github.com/PrincetonUniversityLibrary/video_reserves). 
+
+### Product Owners
+Our Resource Sharing staff are the primary staff users of Ares. Refer any functional questions about the application to them. 
+
+### Web Interface
+Atlas hosts the web interface code in a [Github repot](https://github.com/AtlasSystems/hosting-ares-princeton). You need to request access to this via Atlas support. [Documentation](https://support.atlas-sys.com/hc/en-us/articles/4407504126611-Editing-Atlas-Hosted-Aeon-Web-Pages-in-GitHub) on the workflow to allow PUL users to submit updates is available. Students, Faculty, or University staff who support the management of course reading lists typically do not log into the Ares web interface but rather interact with it through the Canvas integration. Resource sharing handle the creation of accounts to use the web interface directly by request on for those auditing courses who do not posses a Princeton net ID. 
```

Comment:
I'm confused here. Atlas is the company that hosts these services for us. And Aeon is mentioned as part of Ares and Tigris . . . what is Aeon, exactly? Is it a software package that we use with different add-ons for two different services? Or is some of this copy-pasta?

Reply:
Atlas hosts both Ares and Aeon for us. They are completely separate applications as is Tigris. I put them all together in a single documentation file since they are hosted. If it's less confusing I can make a separate file each application.